python/amici/petab_objective.py

```diff
@@ -155,6 +159,114 @@ def simulate_petab(
     }
 
 
+def export_steadystates(
+        petab_problem: petab.Problem,
+        amici_model: AmiciModel,
+        solver: Optional[amici.Solver] = None,
+        problem_parameters: Optional[Dict[str, float]] = None,
+        simulation_conditions: Union[pd.DataFrame, Dict] = None,
+        edatas: List[AmiciExpData] = None,
+        parameter_mapping: ParameterMapping = None,
+        scaled_parameters: Optional[bool] = False,
+        log_level: int = logging.WARNING,
+        num_threads: int = 1,
+        export_file: str = None,
+        sep: str = '\t'
+) -> pd.DataFrame:
+    """Simulate PEtab model.
+    :param petab_problem:
+        PEtab problem to work on.
+    :param amici_model:
+        AMICI Model assumed to be compatible with ``petab_problem``.
+    :param solver:
+        An AMICI solver. Will use default options if None.
+    :param problem_parameters:
+        Run simulation with these parameters. If None, PEtab `nominalValues`
+        will be used). To be provided as dict,  mapping PEtab problem
+        parameters to SBML IDs.
+    :param simulation_conditions:
+        Result of `petab.get_simulation_conditions`. Can be provided to save
+        time if this has be obtained before.
+        Not required if `edatas` and `parameter_mapping` are provided.
+    :param edatas:
+        Experimental data. Parameters are inserted in-place for simulation.
+    :param parameter_mapping:
+        Optional precomputed PEtab parameter mapping for efficiency, as
+        generated by `create_parameter_mapping`.
+    :param scaled_parameters:
+        If True, problem_parameters are assumed to be on the scale provided
+        in the PEtab parameter table and will be unscaled. If False, they
+        are assumed to be in linear scale.
+    :param log_level:
+        Log level, see :mod:`amici.logging` module.
+    :num_threads:
+        Number of cores to be used for simulations when multi-threading
+    :export_file:
+        Path to (csv) file to which the steady state value should be written
+    :sep:
+        Separator for csv file for export
+    :return:
+        Pandas Dataframe containing
+        * the names of the state variables as row indices,
+        * the indices of the amici edata objects as columns,
+          (**NOTE**: only edatas with timepoints INF will be considered)
+        * the steady state values of the state variables as table content`.
+    """
+
+    # if steady state measurements are available: get the simulation conditions
+    steadystate_conditions = _find_steadystate_conditions(petab_problem)
+
+    if simulation_conditions is not None:
+        # filter simulation conditions for steadystate simulations
+        simulation_conditions = _filter_simulation_conditions(
+            simulation_conditions, steadystate_conditions)
```

Comment:
I am not completely sure if understand the overall purpose of the function, but why not provide steadystates for all simulation conditions, independently of whether there are steadystate measurements?